pytest_tests/helpers/s3_helper.py

```diff
@@ -12,6 +12,37 @@
 logger = logging.getLogger("NeoLogger")
 
 
+ACL_TO_PERMISSION_MAP_BUCKET = {
+    "Group": {
+        "public-read-write": "FULL_CONTROL",
+        "grant-full-control": "FULL_CONTROL",
+        "public-read": "READ",
+        "grant-read": "READ",
+        "authenticated-read": "READ",
+        "grant-write": "WRITE",
+    },
+    "CanonicalUser": {
+        "public-read-write": "FULL_CONTROL",
+        "grant-full-control": "FULL_CONTROL",
+        "public-read": "FULL_CONTROL",
+        "grant-read": "FULL_CONTROL",
+        "authenticated-read": "FULL_CONTROL",
+        "grant-write": "FULL_CONTROL",
+        "private": "FULL_CONTROL",
+    },
+}
+
+ACL_TO_PERMISSION_MAP_OBJECT = {
+    "Group": {"public-read-write": "WRITE", "public-read": "READ", "grant-read": "READ"},
+    "CanonicalUser": {
+        "public-read-write": "FULL_CONTROL",
+        "public-read": "WRITE",
+        "grant-read": "WRITE",
+        "private": "WRITE",
```

Comment:
AWS documentation is so messy, but according to [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html), I tend to think, these rows should be FULL_CONTROL as well

Reply:
> AWS documentation is so messy, but according to [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html), I tend to think, these rows should be FULL_CONTROL as well

Then you need to fix that in the s3-gw repo too

Reply:
That's the way. But I think we should discuss it with @evgeniiz321, to be sure we are on the same page with documentation understanding.

Reply:
> AWS documentation is so messy, but according to [Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html), I tend to think, these rows should be FULL_CONTROL as well

Fixed